src/app/(main)/my-voyage/[teamId]/ideation/finalize/components/FinalizeIdeationList.tsx

```diff
@@ -0,0 +1,58 @@
+"use client";
+
+import { useEffect, useState } from "react";
+import { useParams, useRouter } from "next/navigation";
+import FinalizeIdeationItem from "./FinalizeIdeationItem";
+import Button from "@/components/Button";
+import { useIdeation } from "@/store/hooks";
+import routePaths from "@/utils/routePaths";
+
+export default function FinalizeIdeationList() {
+  const { projectIdeas } = useIdeation();
+  const [finalizedIdeation, setFinalizedIdeation] = useState("");
+  const router = useRouter();
+  const { teamId } = useParams<{ teamId: string }>();
+
+  function handleCancelClick() {
+    router.back();
+  }
+
+  useEffect(() => {
+    if (projectIdeas.length === 0) {
+      router.push(routePaths.ideationPage(teamId));
+    }
+  }, [projectIdeas, router, teamId]);
+
+  return (
+    <div className="w-[871px]">
```

Comment:
Again to make it responsive: instead of `w-[871px]` you can use `max-w-[871px] w-full`

Also in`div` below `py-10` need to be changed to `p-10`